packages/webapp/src/__mock-data__/test-developer-application.js

```diff
@@ -78,220 +78,72 @@ const data = `{
-        "Length (km)": 0.3,
-        "Strategic significance": "Area/compensation not in local strategy/ no local strategy",
-        "Total hedgerow units": 0.6,
-        "Off-site reference": "BGS-111 222 333",
-        "Condition": "Poor"
-      }
-    ],
-    "e2": [
-      {
-        "Habitat type": "Native hedgerow with trees",
-        "Length (km)": 0.3,
-        "Delay in starting habitat creation (years)": 0,
-        "Off-site reference": "BGS-111 222 333",
-        "Hedge units delivered": 1.7654229486,
-        "Condition": "Good",
-        "Habitat created in advance (years)": 0
-      }
-    ],
-    "e3": [
-      {
-        "Baseline habitat": "Native hedgerow - associated with bank or ditch",
-        "Length (km)": 0.3,
-        "Habitat enhanced in advance (years)": 0,
-        "Off-site reference": "BGS-111 222 333",
-        "Hedge units delivered": 2.27835855,
-        "Condition": "Moderate"
-      }
-    ],
-    "f1": [
-      {
-        "Watercourse type": "Ditches",
-        "Length (km)": 0.3,
-        "Strategic significance": "Area/compensation not in local strategy/ no local strategy",
-        "Extent of encroachment": "No Encroachment",
-        "Extent of encroachment for both banks": "No Encroachment/ No Encroachment",
-        "Total watercourse units": 1.2,
-        "Off-site reference": "BGS-111 222 333",
-        "Condition": "Poor"
-      },
-      {
-        "Watercourse type": "Ditches",
-        "Length (km)": 0.3,
-        "Strategic significance": "Area/compensation not in local strategy/ no local strategy",
-        "Extent of encroachment": "No Encroachment",
-        "Extent of encroachment for both banks": "No Encroachment/ No Encroachment",
-        "Total watercourse units": 1.2,
-        "Off-site reference": "BGS-111 222 333",
-        "Condition": "Poor"
-      }
-    ],
-    "f2": [
-      {
-        "Habitat created in advance (years)": 0,
-        "Delay in starting habitat creation (years)": 0,
-        "Extent of encroachment for both banks": "No Encroachment/ No Encroachment",
-        "Off-site reference": "BGS-111 222 333",
-        "Watercourse units delivered": 2.594403979575,
-        "Condition": "Fairly Good",
-        "Length (km)": 0.3
-      }
-    ],
-    "f3": [
-      {
-        "Baseline habitat": "Ditches",
-        "Length (km)": 0.3,
-        "Habitat enhanced in advance (years)": 0,
-        "Delay in starting habitat enhancement (years)": 0,
-        "Extent of encroachment": "No Encroachment",
-        "Extent of encroachment for both banks": "No Encroachment/ No Encroachment",
-        "Off-site reference": "BGS-111 222 333",
-        "Proposed habitat": "Ditches",
-        "Watercourse units delivered": 2.409217854828,
-        "Condition": "Good"
-      }
-    ],
-    "validation": {
-      "isMetricWorkbook": true,
-      "isSupportedVersion": true,
-      "isOffsiteDataPresent": true,
-      "areOffsiteTotalsCorrect": {}
+  "gainSite": {
```

Comment:
I think these changes to this file will break the seed data. This file should contain the session data so should contain the raw data returned from the metric extraction into`"developer-metric-data"`.

So it needs to be updated like you have done for `__mocks__/developer-application-data.js` below rather than replacing metric data.

Here you seem to have replaced it with the payload data format instead, which isn't right.

Reply:
yes mock data fixed